src/lib.rs

```diff
@@ -284,6 +284,7 @@ use std::ops::{
     Add, AddAssign, BitAnd, BitAndAssign, BitOr, BitOrAssign, Deref, DerefMut, Index, IndexMut,
     Sub, SubAssign,
 };
+//mod unit_tests;
```

Comment:
Presumably you meant

```suggestion
#[cfg(test)]
mod unit_tests;
```